OWNERS

```diff
@@ -4,6 +4,7 @@ approvers:
 - serathius
 - dgrisonnet
 - logicalhan
+- CatherineF-dev # only for v0.7 release, will remove after v0.7 release 
```

Comment:
This should only have code merge privileges and should not complete the release. need to be in [the list](https://github.com/kubernetes/k8s.io/blob/46695551d62db356ecf1d81b9c1f4a44b8dd2a9b/groups/sig-instrumentation/groups.yaml#L33C1-L43) to push image to staging and select hash of image to [release](https://github.com/kubernetes/k8s.io/blob/46695551d62db356ecf1d81b9c1f4a44b8dd2a9b/registry.k8s.io/images/k8s-staging-metrics-server/images.yaml#L4)

Reply:
Got it, thx!